packages/manager/modules/cloud-connect/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@ovh-ux/manager-cloud-connect",
+  "version": "0.0.0",
+  "private": true,
+  "description": "OVHcloud CloudConnect product",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/modules/cloud-connect"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "main": "./src/index.js",
+  "dependencies": {
+    "lodash": "^4.17.14"
+  },
+  "peerDependencies": {
+    "@ovh-ux/manager-core": "^9.0.0 || ^10.0.0",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.5.0",
```

Comment:
Can't we go without this module ?

Reply:
We are using this to messages too to display in each route.